skpro/regression/cyclic_boosting.py

```diff
@@ -0,0 +1,315 @@
+        "capability:multioutput": False,
+        "capability:missing": True,
+        "X_inner_mtype": "pd_DataFrame_Table",
+        "y_inner_mtype": "pd_DataFrame_Table",
+        "python_dependencies": "cyclic_boosting>=1.2.1",
+    }
+
+    def __init__(self, feature_properties, interaction=tuple(), distr_type="Normal"):
+        self.feature_properties = feature_properties
+        self.interaction = interaction
+        self.quantiles = [0.2, 0.5, 0.8]
+        self.quantile_values = []
+        self.quantile_est = []
+        self.qpd = None
+        self.distr_type = distr_type
+
+        super().__init__()
+
+        # check parameters
+        if not isinstance(feature_properties, dict):
+            raise ValueError("feature_properties must be dict")
+        for i in interaction:
+            if not isinstance(i, tuple):
+                raise ValueError("interaction must be tuple")
+
+        # build estimators
+        features = list(self.feature_properties.keys())
+        for i in interaction:
+            features.append(i)
+
+        for quantile in self.quantiles:
+            self.quantile_est.append(
+                pipeline_CBMultiplicativeQuantileRegressor(
+                    quantile=quantile,
+                    feature_properties=self.feature_properties,
+                    feature_groups=features,
+                    maximal_iterations=50,
+                )
+            )
+
+    def _fit(self, X, y):
+        """Fit regressor to training data.
+
+        Writes to self:
+            Sets fitted model attributes ending in "_".
+
+        Parameters
+        ----------
+        X : pandas DataFrame
+            feature instances to fit regressor to
+        y : pandas DataFrame, must be same length as X
+            labels to fit regressor to
+
+        Returns
+        -------
+        self : reference to self
+        """
+
+        self._y_cols = y.columns
+        y = y.to_numpy().flatten()
+
+        # multiple quantile regression for full probability estimation
+        for est in self.quantile_est:
+            est.fit(X.copy(), y)
+
+        return self
+
+    def _predict(self, X):
+        """Predict labels for data from features.
+
+        State required:
+            Requires state to be "fitted" = self.is_fitted=True
+
+        Accesses in self:
+            Fitted model attributes ending in "_"
+
+        Parameters
+        ----------
+        X : pandas DataFrame, must have same columns as X in `fit`
+            data to predict labels for
+
+        Returns
+        -------
+        y : pandas DataFrame, same length as `X`, same columns as `y` in `fit`
+            labels predicted for `X`
+        """
+
+        index = X.index
+        y_cols = self._y_cols
+
+        # quantile prediction
+        for est in self.quantile_est:
+            yhat = est.predict(X.copy())
+            self.quantile_values.append(yhat)
+
+        j_qpd = J_QPD_S(
+            0.2,
```

Comment:
I would suggest to make this a parameter of the estimator, 0.2 is an arbitrary choice.